src/app/containers/Video/helpers/mediatorUrl.js

```diff
@@ -0,0 +1,4 @@
+const mediatorURL = env =>
+  env === 'test' || env === 'local' ? 'open.test.bbc.co.uk' : 'open.bbc.co.uk';
```

Comment:
```suggestion
  env === 'live' ? 'open.bbc.co.uk' : 'open.test.bbc.co.uk';
```
imo we should check for `live` else default to `test`

Reply:
@sareh said it's safer editorially to fall back to live if there's no env?

Reply:
In what instance would there be no env?